workshop/2 - RWA Overview/README.MD

```diff
@@ -0,0 +1,49 @@
+# 2 - Reliable Web App Pattern - .NET Overview
+
+The reliable web app pattern (or RWA) is a set of principles that helps developers successfully migrate web applications to the cloud. It provides implementation guidance built on the Azure Well-Architected Framework. The pattern focuses on the minimal changes you need to make to ensure the success of your web app in the cloud.
+
+In this workshop series, we're going to run through the principles and show how to apply them to your web applications.
+
+## Enterprise web app cloud journey
+
+Imagine a web application's evolution in the cloud as a journey. Each step in the journey has different goals. The reliable web app pattern is designed to help you take the first step in the journey. You may only need to "lift and shift" or replatform your application to the cloud, and do so with a minimal amount of changes. The reliable web app pattern helps you do that.
+
+![Enterprise web app cloud journey](../images/2-RWA%20Overview/enterprise-web-app-cloud-journey.png)
+
+Further along in the journey, you may want to refactor your application to take advantage of cloud-native features and then optimize it for scale and performance.
+
+But first, let's explore what it means to converge in the cloud and create an application that's reliable with minimal changes in your existing investment.
```

Comment:
... and replatform an application to the Azure cloud with minimal changes but also enhances the reliability and security of the application.